### ISSUE TITLE
Siti per PA

### ISSUE THREAD

**Issue by 260271tatiana** [w3c/wcag#3512]
Il template messo a disposizione dal miur per le PA (scuole) può ritenersi completamente conforme riguardo ai requisiti di accessibilità previsti dalle WCAG 2.1? 
Nel caso siano pubblicati contenuti come foto e video non completamente accessibili il sito continua a rispettare i requisiti di accessibilità?

**Comment by patrickhlauke** [comment]
credo che non ci siano troppi esperti italiani qui ... avresti una referenza o un link per il template?

nel caso di foto/video inaccessibili ... il sito non sara' accessibile, no...

**Comment by rscano** [comment]
Ciao, è stato sviluppato per quello, penso abbiano fatto dei test a riguardo. Segnalo il post al team che ha sviluppato così ti sapranno rispondere :)

**Comment by 260271tatiana** [comment]
Il sito realizzato col template del ministero dell'istruzione italiana è raggiungibile al seguente indirizzo https://www.iclequile.edu.it/ 
Sicvome sono stati pubblicati documenti, circolari etc vorrei sapere se continua ad essere conforme oppure no alle wcag2.1 (fatemi sapere se avete esperienza. A me sembra che alcuni requisiti sulla conformità non siano più rispettati)

**Comment by 260271tatiana** [comment]
> credo che non ci siano troppi esperti italiani qui ... avresti una referenza o un link per il template?
> 
> nel caso di foto/video inaccessibili ... il sito non sara' accessibile, no...

Il sito realizzato col template del ministero dell'istruzione italiana è raggiungibile al seguente indirizzo https://www.iclequile.edu.it/
Sicvome sono stati pubblicati documenti, circolari etc vorrei sapere se continua ad essere conforme oppure no alle wcag2.1 (fatemi sapere se avete esperienza. A me sembra che alcuni requisiti sulla conformità non siano più rispettati)

**Comment by rscano** [comment]
L'eventuale conformità è relativa al modello. Qualsiasi contenuto successivo è responsabilità di chi lo implementa.

**Comment by 260271tatiana** [comment]
In parte ho pubblicato io seguendo le linee guida volevo però un riscontro da qualcuno che ne sapesse più di me. Comunque grazie per le vostre risposte.

**Comment by danielenole** [comment]
Ciao @patrickhlauke @rscano @260271tatiana, il modello Scuole è stato progettato per rispettare i requisiti di accessibilità per le interfacce e quindi da un punto di vista di layout e interazione con le schermate del sito. Essendo un modello di sito (quindi un template non applicato a un caso specifico), la cura dell'accessibilità dei contenuti spetta ai singoli enti che lo implementano: dalla qualità dei contenuti multimediali, alla presenza di alt-text, eventuale sottotitoli video, uso di linguaggio accessibile, uso sporadico di grassetto, corsivo etc. Per ricevere risposte più puntuali sul modello scuole, consiglio di usare i canali predisposti: il [repo github dei template HTML](https://github.com/italia/design-scuole-pagine-statiche/), il [repo github del tema wordpress](https://github.com/italia/design-scuole-wordpress-theme) e il canale #design-siti-scuole dello [Slack Developers Italia](https://join.slack.com/t/developersitalia/shared_invite/zt-26acztaip-_j45jD~).

**Comment by 260271tatiana** [comment]
Ti ringrazio moltissimo. Mi affiderò ai tuoi consigli. 👋
Tatiana Vantaggio

Il Mar 31 Ott 2023, 16:14 Daniele Nolè ***@***.***> ha
scritto:
